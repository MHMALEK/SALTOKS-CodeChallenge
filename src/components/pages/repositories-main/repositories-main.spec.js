import React from 'react';
import renderer from 'react-test-renderer';
import {MemoryRouter} from 'react-router-dom';
import RepositoriesMainPage from './repositories-main-page';
describe('Not Found page Snappshot ', () => {
  test('repos main page renders', () => {
    const component = renderer.create(
        <MemoryRouter>
          <RepositoriesMainPage />
        </MemoryRouter>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
