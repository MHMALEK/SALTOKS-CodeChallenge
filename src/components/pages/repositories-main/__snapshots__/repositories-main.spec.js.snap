// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found page Snappshot  repos main page renders 1`] = `null`;
